DigitalLearningSolutions.Data/Services/JobGroupsService.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using Dapper;
+    using Microsoft.Extensions.Logging;
+
+    public interface IJobGroupsService
+    {
+        string GetJobGroupName(int jobGroupId);
+        IEnumerable<(int, string)> GetJobGroups();
+    }
+
+    public class JobGroupsService : IJobGroupsService
+    {
+        private readonly IDbConnection connection;
+        private readonly ILogger<JobGroupsService> logger;
+
+        public JobGroupsService(IDbConnection connection, ILogger<JobGroupsService> logger)
+        {
+            this.connection = connection;
+            this.logger = logger;
+        }
+
+        public string GetJobGroupName(int jobGroupId)
+        {
+            var name = connection.QueryFirstOrDefault<string?>(
+                @"SELECT JobGroupName
+                        FROM JobGroups
+                        WHERE JobGroupID = @jobGroupId",
+                new { jobGroupId }
+            );
+            if (name == null)
+            {
+                logger.LogWarning(
+                    $"No centre found for job group id {jobGroupId}"
+                );
+            }
+
+            return name;

```

Comment:
Given that this returns the name even if it's null, should we make the return value nullable? Or are we always expecting the name to exist and are happy if it throws if that's not the case?

Reply:
I just forgot to make this one nullable, nice catch